docs/_blog/2024_q4_code_audit.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "2024 Q4 Code Audit"
+description: "2024 Code Audit by Wizardsardine"
+authors:
+ - Steve Myers
+date: "2024-12-03"
+tags: ["BDK", "project"]
+draft: false
+---
+
+A heartfelt thank you to our friends at [Spiral](https://spiral.xyz/) for sponsoring a code audit of the current `bdk` 1.0.0-beta Rust codebase. The effort was led by [Antoine Poinsot](https://gist.github.com/darosior) from [Wizardsardine](https://wizardsardine.com/), who did a fantastic job provided insightful and actionable recommendations for the BDK team. You can find the full report [here](https://gist.github.com/darosior/4aeb9512d7f1ac7666abc317d6f9453b).
```

Comment:
...who did a fantastic job _provided_... `---->` ...who did a fantastic job _providing_...?

Reply:
Thanks for catching my grammar mistake, English is my native language but I'm not that good at it.😆